recipes/lapy/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lapy" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lapy-{{ version }}.tar.gz
+  sha256: a3778ac2d7e8f22e9499e46c590408ae3236f9703b7782840d451832c7e9ccef
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - lapy-sys_info = lapy.commands.sys_info:run
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools >=61.0.0
+  run:
+    - nibabel
+    - numpy >=1.22
+    - plotly
+    - psutil
+    - python >=3.8
+    - scikit-sparse
+    - scipy
+
+test:
+  imports:
+    - lapy
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - lapy-sys_info --help
```
This just tests the CLI